ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -429,6 +429,90 @@ def can_be_revised(config: Config, db_config: SimpleConnectionPool, entry: dict[
     return True
 
 
+def no_change(config: Config, db_config: SimpleConnectionPool, entry: dict[str, Any]) -> bool:
```

Comment:
Maybe name could be slightly more self-documenting :D 

`is_flatfile_data_changed`